plutus-pab/tx-inject/Main.hs

```diff
@@ -84,7 +87,7 @@ runProducer AppEnv{txQueue, stats, utxoIndex} = do
   where
     producer :: GenIO -> UtxoIndex -> IO ()
     producer rng utxo = do
-      tx <- generateTx rng utxo
+      tx <- generateTx rng (Slot 100) utxo
```

Comment:
Why `Slot 100`?

Reply:
We kind of need a slot in order to be able to validate the transaction. In our case the slot number itself, does not really matter as we don't use transactions with slot bounds in tests. I will switch to slot 42 and add a comment.